app/src/main/java/com/saotome/imed_app/ui/fragments/EditDrug.kt

```diff
@@ -0,0 +1,60 @@
+package com.saotome.imed_app.ui.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.saotome.imed_app.R
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
```

Comment:
Was all this code for handling arguments added when you created the fragment?

Reply:
Yes, this was all auto generated. I created the new fragment with its companion class just to have the structure ready. I haven't touched it yet - will do it on my next PR.